composites/Plugin/ContentAnalysis/components/AnalysisResult.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { injectIntl, intlShape, defineMessages } from "react-intl";
+
+import { circle } from "../../../../style-guide/svg/index";
+import { eye } from "../../../../style-guide/svg/index";
+import { IconWithAriaLabel } from "../../Shared/components/Icon.js";
+import IconButtonToggle from "../../Shared/components/IconButtonToggle.js";
+
+const messages = defineMessages( {
+	SEOScore: {
+		id: "analysis.SEOScore",
+		defaultMessage: "SEO score",
+	},
+} );
+
+/**
+ * Returns a AnalysisResultBase component.
+ *
+ * @param {object} props Component props.
+ *
+ * @returns {ReactElement} The rendered AnalysisResultBase component.
+ */
+const AnalysisResultBase = styled.li`
```

Comment:
As far as I know, we're not documenting styled components. Should this docblock be removed?